app/styles/govflanders/components/data-table/_actions.scss

```diff
@@ -1,4 +1,10 @@
 .vl-data-table {
+
+  // override the default data-table line.
+  .data-table__actions--top{
+    border-bottom:0px solid transparent !important;
```

Comment:
This would be better. We no units when 0, and some spacing. But when unsetting a border it's better to just declare it as `none`.
```css
border-bottom: none !important;
```

Reply:
bedankt, zo leer ik ook weer iets bij